sveltekit/src/app.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8" />
+		<link rel="icon" href="%sveltekit.assets%/favicon.png" />
+		<meta name="viewport" content="width=device-width" />
+		%sveltekit.head%
+	</head>
+	<body data-sveltekit-preload-data="hover">
```

Comment:
Small tweak to remove the margin around the page.
```suggestion
	<body data-sveltekit-preload-data="hover" style="margin:0px">
```